sdk/core/System.ServiceModel.Rest/src/RequestBody.cs

```diff
@@ -34,5 +43,71 @@ public abstract class RequestBody : IDisposable
 
         /// <inheritdoc/>
         public abstract void Dispose();
+
+        // TODO: Note, this is copied from RequestContent.  When we can remove the corresponding
```

Comment:
Additional note on clean up during unification per https://github.com/Azure/azure-sdk-for-net/pull/38860/files#r1333559915